scripts/bert/finetune_classifier.py

```diff
@@ -110,6 +109,10 @@
     type=int,
     default=128,
     help='Maximum length of the sentence pairs, default is 128')
+parser.add_argument(
+    '--pad',
+    action='store_true',
+    help='whether to do padding when pre-processing dataset. Default is False.')
```

Comment:
"Whether to pad to maximum length when preparing data batches. Default is False."

Reply:
Fixed.